dvc/utils/pkg.py

```diff
@@ -1,63 +1,7 @@
-from dvc.utils import is_binary
-
-
-def is_conda():
+def get_package_manager():
```

Comment:
Btw, no good reason to make this dynamic, let's just:
```
try:
    from .build import PKG  # file created during dvc build
except ImportError:
    PKG = None
```

and then just import PKG where needed, instead of `get_package_manager`

Reply:
DeepSource shows that as anti-pattern, do you still want to keep without function wrap?

Reply:
@n3hrox DS is saying that PKG is unused, which is on purpose here, so we are fine :slightly_smiling_face: